src/explorer/Components/EmailCheckComponent.cs

```diff
@@ -25,14 +26,24 @@ public async IAsyncEnumerable<ExploreMetric> YieldMetrics()
             yield return new UntypedMetric(name: "is_email", metric: await ResultAsync);
         }
 
-        protected override Task<bool> Explore() => CheckIsEmail(conn, ctx);
+        protected override Task<Result> Explore() => CheckIsEmail(conn, ctx);
 
-        private static async Task<bool> CheckIsEmail(DConnection conn, ExplorerContext ctx)
+        private static async Task<Result> CheckIsEmail(DConnection conn, ExplorerContext ctx)
         {
             var emailCheck = await conn.Exec(
                 new TextColumnTrim(ctx.Table, ctx.Column, TextColumnTrimType.Both, Constants.EmailAddressChars));
+            var isEmail = emailCheck.Rows.All(r => r.IsNull || (!r.IsSuppressed && r.Value == "@"));
+            return new Result(isEmail);
+        }
+
+        public class Result
+        {
+            public Result(bool value)
+            {
+                Value = value;
+            }
 
-            return emailCheck.Rows.All(r => r.IsNull || (!r.IsSuppressed && r.Value == "@"));
+            public bool Value { get; }
```

Comment:
I would suggest renaming this to something more expressive, for example `IsEmail`

Reply:
fixed with 0a5f8817de21fb8a60dc87e62352060aa6158278